src/engine/effects/groupfeaturestate.h

```diff
@@ -10,7 +10,10 @@ struct GroupFeatureState {
               has_beat_fraction(false),
               beat_fraction(0.0),
               has_gain(false),
-              gain(1.0) {
+              gain(1.0),
+              has_bpm(false),
+              bpm(0.0),
```

Comment:
I think I should set the default bpm to 120 and not zero.

Reply:
Why 120? Maybe the "other" bpm as if the track where synced.

Reply:
Yes, We can do that. But actually setting a bpm here won't make a difference in my opinion because anyways the has_bpm boolean will be false and the metronome effect can sync it with the master clock in that case.